### ISSUE TITLE
Splits for Gene Expression Prediction (Table 6 in the paper)

### ISSUE THREAD

**Issue by DhruvaRajwade** [peng-lab/hescape#12]
Thank you for the great benchmark and evaluation. For Table 6 in the paper, where metrics are reported as Mean(std), I was wondering how I could reproduce the std. Were these runs across different (and every) patient(s) as the test set, or using the fixed splits given in `hescape/data/any_dataset/test.csv` but with resampling of the test spots?

**Comment by rushin682** [comment]
Hi,
Hest benchmark (used for table 6) as provided with the HEST framework requires a fine-tuning step. This was done in a cross-validation setting on the test.csv. Using the test set is crucial since the hescape models have already seen the train and val sets during the pretraining step. 

The mean and std is based on a cross-validation split of the test.csv. Make sure you're using the HEST-benchmark config (yaml file) as suggested by the authors of HEST (also mentioned below):

```
# directory containing the data for each task
bench_data_root: <hest datasets used for the task>

# directory where benchmark results will be dumped
results_dir: <directory path>

# directory where the vision embeddings will be dumped
embed_dataroot: <directory path>

# directory to the model weights root
weights_root: <directory path>

# inference parameters
batch_size: 128
num_workers: 1

# encoders to benchmark
encoders: [
    # OG models
    # "uni_v1", # uncomment after requesting the weights
    #"resnet50",
    # "ctranspath",
    #"phikon",
    #"phikon_v2",
    # "h0_mini", # uncomment after requesting the weights
    #"hibou_large",
    #"kaiko_base_8",
    #"remedis", # uncomment after requesting the weights
    # "conch_v1", # uncomment after requesting the weights
    # "gigapath", # uncomment after requesting the weights
    #"virchow", # uncomment after requesting the weights
    #"virchow2", # uncomment after requesting the weights
    #"hoptimus0",

    # HESCAPE models
    "hescape_multi_tissue_h0_mini",
    "hescape_multi_tissue_gigapath",
    "hescape_multi_tissue_uni",
    "hescape_5k_h0_mini",
    "hescape_5k_gigapath",
    "hescape_5k_uni",
  ]

# datasets contained in `bench_data_root` to benchmark
datasets: [
    # works
    "human_breast_panel",
    "human_immuno_oncology_panel",
    "human_5k_panel",
    "human_lung_healthy_panel",
  ]
#dimreduce: "PCA"
latent_dim: 128
```

Feel free to open this issue again if you have questions.
